WalletWasabi/BitcoinP2p/TrustedP2pBehavior.cs

```diff
@@ -46,6 +50,44 @@ protected override bool ProcessInventoryVector(InventoryVector inv, EndPoint rem
 			return false;
 		}
 
+		protected override void ProcessTx(TxPayload payload)
+		{
+			var txid = payload.Object.GetHash();
+			bool toProcess;
+
+			lock (DisabledFromProcessingLock)
+			{
+				toProcess = !DisabledFromProcessing.Contains(txid);
```

Comment:
Add some comment what is this about?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/5728/commits/47c91f3421cc236e390c5d2aa1f8a9d3e1ce1c1d